public/style.css

```diff
@@ -51,42 +62,55 @@ button:focus {
 
 .talent-nav {
   position: absolute;
-  list-style-type: none;
-  top: 0;
-  right: 10rem;
+  clear: both;
+  max-height: 0;
+  top: 8.5rem;
+  left: -5rem;
+  width: fit-content;
 }
 
 .talent-nav ul {
-  list-style-type: none;
   margin: 0;
   padding: 0;
+  list-style: none;
+  overflow: hidden;
 }
 
 .talent-nav li {
-  float: left;
+  position: relative;
+  display: block;
+  background-color: rgb(248, 248, 248);
+  left: -1.75rem;
 }
 
 .talent-nav li a {
   display: -webkit-box;
   text-align: center;
-  padding: 16px;
+  padding: 10px;
```

Comment:
what are the implications of sometimes using px and sometimes using rems? (i'm not going to block the PR on this, but would you leave a comment about why you decided to use px here?)

Reply:
I hadn't noticed that sorry I will change it with the next pull request I make.